app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -13,3 +14,11 @@ export function get(projectId) {
       stories: projectBoard.stories.map(Story.deserialize)
     }));
 };
+
+export const projectBoardPropTypesShape = PropTypes.shape({
+  error: PropTypes.oneOf([
+    PropTypes.object,
+    PropTypes.array
+  ]),
+  isFetched: PropTypes.bool.isRequired
+}).isRequired
```

Comment:
I think this should be where it's used, not here. Otherwise you can't use a optional `projectBoard`, for example (it applies for the other ones as well).